### ISSUE TITLE
[CS 1.6] Feature request - Crosshair Style, toggle 

### ISSUE THREAD

**Issue by i486** [ValveSoftware/halflife#2105]
I know it's a long shot, but it would be really cool if Valve could add some simple CSGO goodies to CS 1.6

Off the top of my head, I would like to see the "toggle" command and Crosshairstyles in CS 1.6

I know Valve is going to work on fixing some CS 1.6 [0day exploits](https://www.zdnet.com/article/malicious-counter-strike-1-6-servers-used-zero-days-to-infect-users-with-malware/), why not surprise us CS 1.6 hardcore fans at the same with the bugfix? :)

**Comment by Pokl1** [comment]
HUD_Color in RGB format would be appreciated

**Comment by Karaulov** [comment]
Try this hack https://gamebanana.com/tools/6478

**Comment by Karaulov** [comment]
Yes some server can be inject from html malware (save bat file to autorun folder, or run bat file, i don't know)

https://forum.kaspersky.com/index.php?/topic/390291-kaspersky-free-%D0%BD%D0%B5-%D1%81%D0%BB%D0%BE%D0%B2%D0%B8%D0%BB-%D0%B2%D0%B8%D1%80%D1%83%D1%81/&do=findComment&comment=2785444

**Comment by i486** [comment]
https://github.com/ValveSoftware/halflife/issues/2255
https://github.com/ValveSoftware/halflife/issues/2468
https://github.com/ValveSoftware/halflife/issues/1188
https://github.com/ValveSoftware/halflife/issues/416
https://github.com/ValveSoftware/halflife/issues/2305
https://github.com/ValveSoftware/halflife/issues/265
https://github.com/ValveSoftware/halflife/issues/1782
https://github.com/ValveSoftware/halflife/issues/1925

Please add your crosshair related requests and suggestions in this threads as a comment since all other crosshair related requests are closed in favor of this one.

**Comment by Automatics** [comment]
please add crosshair outline or other color blind options.

**Comment by Automatics** [comment]
https://www.youtube.com/watch?v=q-nsio1Jj_E

**Comment by ashtonhogan** [comment]
Go play csgo?

**Comment by Automatics** [comment]
why do you care what OPTIONAL settings are included? is someone forcing you to use these non existant settings oh forum troll?

**Comment by ashtonhogan** [comment]
By that logic, let's just make an option to play cs go within 1.6, why stop at the crosshair?

**Comment by Maxi605** [comment]
So what, since the option is on CS:GO it can't be anywhere else? if i want that option i should only limit myself to play CS:GO?

Your way of thinking is absurd. I like this idea, would be pretty good to have more options of customization of the crosshair.

**Comment by ashtonhogan** [comment]
Yes that's right!
Go play cs go!
